### PR TITLE
Dissolved /enterprise/ dir, made TOC titles match top-of-page title

### PR DESCRIPTION
**Pull request by mikehoffms**
**The four moved .md files/pages, rendered, in order of existing TOC:**
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/declare-permissions?branch=pr-en-us-1985 - resided in /enterprise/ dir but was presented in Developer Guide TOC bucket & remains there.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/match-patterns?branch=pr-en-us-1985 - moved to Developer Guide TOC bucket, **OK?**

* These two pages moved from /enterprise/ dir to /publish/ dir, and moved from Enterprise TOC bucket (which is now gone) to Publish TOC bucket next to other Publish articles, **OK?**
   * https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/hosting-and-updating?branch=pr-en-us-1985
   * https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/auto-update?branch=pr-en-us-1985

**The redirects to test:**
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/declare-permissions?branch=pr-en-us-1985 - passed.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/match-patterns?branch=pr-en-us-1985 - passed.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/hosting-and-updating?branch=pr-en-us-1985 - passed.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/auto-update?branch=pr-en-us-1985 - passed.

### CODE REVIEW

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/toc.yml
Comment (mikehoffms) [inline microsoft-edge/toc.yml:853]: @sonakshimore See Note at top of the upstream doc, https://developer.chrome.com/docs/apps/autoupdate/

File: microsoft-edge/toc.yml
Comment (mikehoffms) [inline microsoft-edge/toc.yml:847]: @ashishnirvana @sonakshimore - Dissolve hosting-and-updating.md; merge its content into other Publish article? rendered:  https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/hosting-and-updating?branch=pr-en-us-1985

**Comment by sonakshimore** [comment]
Looks good to me.

@mikehoffms what about the conflicting files?

File: microsoft-edge/extensions-chromium/publish/auto-update.md
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/auto-update.md:26]: we can merge this document for now, but separately we should review this document with engineering team for its relevance.

File: microsoft-edge/toc.yml
Comment (ashishnirvana1) [reply microsoft-edge/toc.yml:850]: Have asked an engineering team mate to review this document. https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/auto-update?branch=pr-en-us-1985

But I recommend we go ahead with merging this - and sharing back the published document link as soon as its merged. We can take his inputs into consideration later.

**Review by ashishnirvana1 — APPROVED** [review]

**Pull request merged**
